### DISCUSSION TITLE
Convert a subtask to an issue is not available anymore

### DISCUSSION THREAD

**Discussion by chubin** [community/community#51960] [Projects and Issues]
### Select Topic Area

Question

### Body

Convert an issue to a subtask is not available anymore.
It worked just a couple of days before, or maybe even today.
It was a very useful feature, and I will miss it a lot.

Is it temporary, or will it be deleted forever?

(I mean this feature https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists)

**Comment by arlindmusliu** [comment]
It's almost a week that it doesn't work for me. I've searched online but there's no solution to this. I've tried different browsers too, with different users, and nothing seems to work. It's such a great feature though.

**Reply by azenMatt** [reply]
Are you still seeing this problem?

**Comment by EnergeticPixels** [comment]
Are you folks referring to the little dotted button that used to be to the right of a subtask item?  I circled where it used to be.
![github_convertToIssue_missing](https://user-images.githubusercontent.com/283558/230494149-b2031fb2-3f39-47db-99b6-48c4daa78b73.JPG)

**Reply by ossiemoore** [reply]
Yes.. This is gone for me too. Really frustrating. This made it so easy to create a (my words) "meta" issue that referenced a collection of other issues.

**Comment by arlindmusliu** [comment]
Yes, that's the one!

**Comment by EnergeticPixels** [comment]
When I got into GH yesterday morning to work on a stretch assignment, I
thought I was going crazy when no matter what I would do, I could not get
the action button to show.  Then I went surfing through GH announcements
thinking they purposely removed the ability and I missed reading about it.
At least I know I am not crazy.
Thank you for the confirmation.
Tony

On Fri, Apr 7, 2023 at 3:49 AM Arlind Musliu ***@***.***>
wrote:

> Yes, that's the one!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/51960#discussioncomment-5551659>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACFHJSAFDBJLE3C3BIBJ6DW77IJDANCNFSM6AAAAAAWS3HL64>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"If you give a monkey a gun, and the monkey shoots someone, do you blame
the monkey or the person who gave the monkey the gun?"
Peggy Bundy

**Comment by arlindmusliu** [comment]
The problem for us was the usage of headings between the task list.

This works:
- [ ] First task
- [ ] Second task
....

This doesn't work:
## Group 1
- [ ] First task
- [ ] Second task
## Group 2
- [ ] First task
- [ ] Second task

**Reply by arlindmusliu** [reply]
It works again for us, even with headings.

**Comment by EnergeticPixels** [comment]
I noticed that.  I did just get in there and the small "Convert to Issue"
action button is back.

Thanks GH for the quick fix.

**Comment by chubin** [comment]
Yes, the problem is fixed. I can confirm it. Thanks Github!

**Comment by navanjr** [comment]
Still not working for me. This is a great feature and I need it, ...please. 😃 
I am testing this in chrome 111 and brave 1.49

Perhaps this is now a paid feature?

**Reply by navanjr** [reply]
fixed as of this morning! thanks

**Comment by chubin** [comment]
No, it is not paid. They have other paid feature (`[tasklist]`), but this one is free.

Show us your markdown

**Reply by navanjr** [reply]
```
### testing Convert task to issue
- [ ] you should be offered the `convert to issue` feature
- [ ] and here as well
```
<img width="908" alt="Screen Shot 2023-04-07 at 12 00 35 PM" src="https://user-images.githubusercontent.com/753288/230648263-be25542a-f6c8-457d-b12d-e03cdb9546c3.png">
** unfortunately the screenshot does not show my mouse cursor

**Reply by chubin** [reply]
I've tested your markdown in my browser (FF) and it seems to work well.
Confirmation from other users is needed (Ideally with Chrome and Brave).

**Reply by navanjr** [reply]
working as of today. thanks for your help!

**Comment by chubin** [comment]
(I reopened the discussion, because some users still have the problem)

**Comment by chubin** [comment]
I noticed that it does not work after "edit". After edit I always have to reload the page (ctrl-r) to make it start working.

**Reply by navanjr** [reply]
reloading (cmd+r) does not change the behavior for me. still no "convert" option on hover

**Reply by navanjr** [reply]
<img width="913" alt="Screen Shot 2023-04-07 at 12 19 44 PM" src="https://user-images.githubusercontent.com/753288/230650546-e9f974f2-bfc7-41ed-923d-a9907bda548e.png">
** again unfortunately the screenshot does show my mouse

**Comment by clshaw01** [comment]
Was also having this problem.  
- Created a comment with task list and convert was not available.
- Edited the list and added the heading, convert still not available.

Created new issue with the task list as the very first comment in the ticket and the task list *is* available.  So I suspect it must exist in the first comment when the issue is created in order to work.

**Reply by sjuliancarrizo** [reply]
I've just experienced the same... So I also believe that the "convert to issue" feature only works in the very first comment of an issue.

**Comment by otech47** [comment]
broken again

![Screenshot 2023-07-24 at 4 15 51 PM](https://github.com/community/community/assets/4914611/18c59236-8270-4623-bb9a-ca5fd237059f)

**Comment by FaGru3n** [comment]
hi all,

sometimes i have this problem: 
![image](https://github.com/community/community/assets/121097161/074bb40c-a778-4116-9502-e6983db127a4)

searched for docs allready, but i could not explain wich permission or even the reason why some users in a GitHub Org can not use the "convert to issue" - feature?

some suggestions why this happens?

**Comment by 03stories** [comment]
I see it working for me on Issue pages, but not on the Issue side panel quick view from a Project view. Should it be working from within a Project?

**Reply by Julian-Dumitrascu** [reply]
I can convert tasks to issues from within a project.

**Reply by iainelder** [reply]
I also have the the same problem as @03stories . Browsing a project using Mozilla Firefox for Ubuntu 128.0 (64-bit).

**Reply by azenMatt** [reply]
We are working on a fix for this at the moment

**Reply by clshaw01** [reply]
Please please please please please x infinity tell me the fix is this: https://github.com/github/roadmap/issues/927

**Reply by azenMatt** [reply]
That is real and is coming soon!
